docs/source/noise/row_noise.rst

```diff
@@ -89,3 +89,48 @@ parameter:
 
 When applying :code:`omit_row` noise, each row of data is selected for omission
 independently with probability :code:`row_probability`.
+
+Duplicate with guardian
+-----------------------
+
+A known challenge in entity resolution is people being reported multiple
+times at different addresses. This can occur when family structures are
+complex and children spend time at multiple households. A related
+challenge occurs with college students, who are sometimes counted both at their
+university and at their guardian’s home address.
+
+To facilitate this type of error, we have simulants assigned to guardians
+within the simulation. Sometimes, those guardians may live at
+different addresses than their dependents. In this case, there is an
+opportunity for duplication. Since this mechanism occurs within the
+simulation, there is a natural maximum on how many simulants can be duplicated
+in this way.
```

Comment:
```suggestion
```
I don't think we need this paragraph from the concept model -- it was targeted more towards the engineers building the sim.